fragments/labels/8x8.sh

```diff
@@ -1,9 +1,12 @@
 8x8)
-    # credit: #D-A-James from MacAdmins Slack and Isaac Ordonez, Mann consulting (@mannconsulting)
```

Comment:
I noticed in Contributing.md that in future comments with credits would be removed to reduce the main script length. I've removed existing comments where possible.